_posts/en/newsletters/2024-09-13-newsletter.md

```diff
@@ -0,0 +1,151 @@
+  Delving Bitcoin to announce a [tool][mutation-core] that would
+  automatically modify (mutate) code changed in a PR or commit to
+  determine whether they will cause existing tests to fail.  Anytime
+  random changes to code doesn't produce a failure indicates that test
+  coverage may be incomplete.  Garcia's automatic mutation tool ignores
+  code comments and other lines of code that wouldn't be expected to
+  produce changes.
+
+- **DLC-based loan contract execution:** Shehzan Maredia
+  [posted][maredia post] Delving Bitcoin to announced [Lava Loans][], a
+  loan contract that uses [DLC][topic dlc] oracles for price discovery
+  of bitcoin-collateralized loans.  For example, Alice offers Bob
+  $100,000 USD if Bob keeps at least 2x $100,000 in BTC in a deposit
+  address.  Oracles trusted by both Alice and Bob periodically publish
+  signatures committing to the current BTC/USD price.  If Bob's BTC
+  collateral drops below the agreed-upon amount, Alice can seize
+  $100,000 USD worth of his BTC at the highest oracle-signed amount.
+  Alternatively, Bob can provide proof onchain that he repaid the loan
+  (in the form of a hash preimage revealed by Alice) to receive back his
+  collateral.  Other resolutions of the contract are available if the
+  parties fail to cooperate or one party becomes unresponsive.  As with
+  any DLC, the price oracles are prevented from learning the contract
+  details or even that their price information is being used in a
+  contract.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting.*
+
+[Testing Bitcoin Core 28.0 Release Candidates][review club
+v28-rc-testing] was a review club meeting that did not review a
+particular PR, but rather was a group testing effort.
+
+Before each [major Bitcoin Core release][], extensive testing by the
+community is considered essential. For this reason, a volunteer writes a
+testing guide for a [release candidate][] so that as many people as
+possible can productively test without having to independently ascertain
+what's new or changed in the release, and reinvent the various setup
+steps to test these features or changes.
+
+Testing can be difficult because when one encounters unexpected
+behavior, it's often unclear if it's due to an actual bug or if the
+tester is making a mistake. It wastes developers' time to report bugs to
+them that aren't real bugs. To mitigate these problems and promote
+testing efforts, a Review Club meeting is held for a particular release
+candidate, in this instance, 28.0rc1.
+
+The [28.0 release candidate testing guide][28.0 testing] was written by
+rkrux, who also hosted the review club meeting.
+
+Attendees were also encouraged to get testing ideas by reading the [28.0
+release notes][].
+
+This review club covered the introduction of [testnet4][topic testnet]
+([Bitcoin Core #29775][]), [TRUC (v3) transactions][topic v3
```

Comment:
I dont have a strong preference on it, but we could link to the Optech PR summaries in this section (vs linking to the PR itself)